en/getting_started/README.md

```diff
@@ -76,7 +77,6 @@ To install Qt:
 1. Install Additional Packages (Platform Specific)
    - **Ubuntu:** `sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev`
    - **Fedora:** `sudo dnf install speech-dispatcher SDL2-devel SDL2 systemd-devel`
-   - **Arch Linux:** `pacman -Sy speech-dispatcher`
```

Comment:
This needs to get put back in

Reply:
Taken care of in https://github.com/mavlink/qgc-dev-guide/pull/124